### ISSUE TITLE
Page navigation buttons cannot be accessed through keyboard focus

### ISSUE THREAD

**Issue by SaschaAdler** [crkn-rcdr/cap#94]
The issue here is that these buttons are actually links, because in the non-JavaScript mode of viewing the site, we need to use link navigation to go from one page to another. I think the current page viewer removes the href attribute of each link, and the browser interprets <a> tags without href attributes as disabled and therefore removes the ability to tab over to them. The solution is to either ensure that the <a> tags still have href attributes, or to finally give up on the idea that non-JavaScript use of this feature makes any sense at all. I’d probably go with the second choice.

**Comment by brittnylapierre** [comment]
Pushed to demo (will create new tickets if needed)
